Userland/Games/Solitaire/Game.cpp

```diff
@@ -466,4 +468,11 @@ void Game::paint_event(GUI::PaintEvent& event)
     }
 }
 
+void Game::dump_layout() const
```

Comment:
This feels like it should be behind something like `SOLITAIRE_DEBUG`

Reply:
Sure, I can add that. My thinking was that the hotkey to trigger this method is a hidden combo, but adding the debug var won't hurt.